### DISCUSSION TITLE
Can't find the network configuration settings GUI package in Arch

### DISCUSSION THREAD

**Discussion by YamiYukiSenpai** [lxqt/lxqt#2298] [Q&A]
What's the package name in Arch to install the network configuration settings in LXQt's Configuration Settings?

**Comment by stefonarch** [comment]
There is no default one, mostly cmst (connman) and nm-tray or nm-applet (networkmanager) are used.

**Reply by YamiYukiSenpai** [reply]
I went for nm-tray/applet, but I'm unable to configure it.  And I don't see the application in Configuration Center

**Reply by tsujan** [reply]
![nm](https://user-images.githubusercontent.com/981076/185800718-fe6890cc-7013-4768-9525-75d1d07a7dcf.png)

**Reply by Butterfly** [reply]
> There is no default one, mostly cmst (connman) and nm-tray or nm-applet (networkmanager) are used.

New LXQt fan as of about 2 months. Just poking my nose in to say terminal CLI-based `wicd-curses` is working fabulously for me on Debian. There's a `wicd` GUI that shows up in Applications menus, but it started having glitches at some point.

Disclaimer: My needs are very basic so there's always that factor as to why `wicd-curses` works well here. It also sometimes takes turning to special repositories to find the packages these days. :+1: :smile:

**Reply by YamiYukiSenpai** [reply]
> ![nm](https://user-images.githubusercontent.com/981076/185800718-fe6890cc-7013-4768-9525-75d1d07a7dcf.png)

I see that on Lubuntu, but not on my newly-installed Arch.

**Reply by stefonarch** [reply]
You have installed nm-applet non nm-tray? I see it fine on arch. Check it's desktop file: `/usr/share/applications/nm-connection-editor.desktop 
`
